gateway/1.4/modules/ROOT/pages/index.adoc

```diff
@@ -1,91 +1,151 @@
+== Flex Gateway Architecture
 
-== Anypoint Flex Gateway
+Flex Gateway contains two components that work in tandem:
 
-Anypoint Flex Gateway is an ultrafast API gateway designed to manage and secure APIs running anywhere.
-Built to seamlessly integrate with DevOps and CI/CD workflows, Anypoint Flex Gateway delivers the
-performance required for the most demanding applications and microservices while providing enterprise
-security and manageability across any environment.
+* The control plane
+* The runtime
 
-For more information, see xref:flex-gateway-getting-started.adoc[Getting Started with Flex Gateway].
+The following graphic diagrams the relationship between these components:
 
-== Anypoint Mule Gateway
+image::gateway-concepts-control-plane-diagram.png["Flex Gateway Concepts diagram,"width=100%]
 
-Mule Runtime includes an embedded Mule Gateway. Using this gateway, any user can, for example, apply a basic authentication policy on top of a Mule application, enrich an incoming/outgoing message, or add any other complex capability to an API without having to write any code.
+=== Control plane
 
-Overall, Mule Gateway allows you to add a dedicated orchestration layer on top of your backend APIs and services to help you separate orchestration from implementation concerns. You can leverage the governance capabilities of API Manager to apply, among other capabilities, throttling, security, caching, and logging to your APIs.
+The control plane is a MuleSoft-hosted platform that centralizes management and observability, enabling you to:
 
-If you want to apply policies and generate analytics info, you must use either:
+* Build APIs.
+* Add policies.
+* Deploy APIs into the runtime.
+* Monitor APIs in Anypoint API Manager.
+* Configure the runtime in Anypoint Runtime Manager.
 
-* An API defined in a Mule application.
-* A Mule application that proxies your existing backend implementation.
+=== Runtime
 
-For more information, see xref:mule-gateway::mule-gateway-capabilities-mule4.adoc[Mule Gateway Capabilities].
+The gateway runtime is a customer-hosted distributed entity installed in any cloud data center or, for testing purposes, on your local laptop. The runtime receives commands from the control plane and routes and protects the backend APIs. Built with security in mind, communications between the runtime and the control plane are secured through mTLS and HTTPs.
+
+FluentBit implementation enables log output to local files, or to aggregators such as New Relic, Sumo Logic, Splunk, among others.
+
+== Replicas
+A runtime unit is called a `replica`. While a single runtime can support multiple backend APIs for high availability, MuleSoft recommends installing Flex Gateway as a cluster - a Kubernetes Ingress controller with multiple replicas running in parallel. Doing so increases gateway performance and robustness, and is recommended for high-performance applications that need to scale.
```

Comment:
Is that an em-dash? I think Kim didn't like those. 

Suggest:
`... as a cluster, which is a .....`

Reply:
Suggest "Using clusters increases..." instead of "Doing so"

Reply:
Kim doesn't like "need to".
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22needto,%22%22haveto,%22%22must%22

Reply:
Fixed